identity-apps-core/apps/authentication-portal/src/main/webapp/enableTOTP.jsp

```diff
@@ -49,10 +49,12 @@
         authenticationFailed = "true";
 
         if (request.getParameter(Constants.AUTH_FAILURE_MSG) != null) {
-            errorMessage = Encode.forHtmlAttribute(request.getParameter(Constants.AUTH_FAILURE_MSG));
+            String error = Encode.forHtmlAttribute(request.getParameter(Constants.AUTH_FAILURE_MSG));
 
             if (errorMessage.equalsIgnoreCase("authentication.fail.message")) {
                 errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle,"error.retry");
+            } else if (!error.equalsIgnoreCase(AuthenticationEndpointUtil.i18n(resourceBundle, error))) {
+                errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, error);
             }
```

Comment:
If matching i18 key found.

<img width="874" height="834" alt="Image" src="https://github.com/user-attachments/assets/0bdaf8ba-7ede-4e6f-b3c9-fa670a89a0c7" />

If matching i18 key not found.

<img width="894" height="847" alt="Image" src="https://github.com/user-attachments/assets/8d75828e-1e11-41c4-a002-5ff575bbf3c6" />